tests/test_other.py

```diff
@@ -11401,3 +11401,18 @@ def test_gmtime_noleak(self):
     # Confirm that gmtime_r does not leak when called in isolation.
     self.emcc_args.append('-fsanitize=leak')
     self.do_other_test('test_gmtime_noleak.c')
+
+  def test_build_fetch_examples(self):
+    # We can't run these outside of the browser, but at least we can
+    # make sure they build.
+    self.set_setting('FETCH')
+    self.build(test_file('fetch/example_synchronous_fetch.c'))
+    self.build(test_file('fetch/example_async_xhr_to_memory.c'))
+    self.build(test_file('fetch/example_async_xhr_to_memory_via_indexeddb.c'))
+    self.build(test_file('fetch/example_idb_delete.c'))
+    self.build(test_file('fetch/example_idb_store.c'))
+    self.build(test_file('fetch/example_stream_async_xhr.c'))
+    self.build(test_file('fetch/example_synchronous_fetch.c'))
+    self.build(test_file('fetch/example_sync_xhr_to_memory.c'))
+    self.build(test_file('fetch/example_waitable_xhr_to_memory.c'))
+    self.build(test_file('fetch/example_xhr_progress.c'))
```

Comment:
If these are tested in the browser suite, why build them here as well? The browser suite is enough in that case I think.

Reply:
They are not tested in the browser suite..  Perhaps one of them is, but most of them are completely untested AFAICT.

Reply:
You can see that they are not (all) tested by that fact there was a variable name typo in a couple of them, right?

Reply:
Ok, how about adding just those that are not tested in the browser suite then? The first one at least has to be tested because it's mentioned in this diff.

Reply:
Sure.. its was only the first one that was used in the browser tests.  Removed it from here.